project/plugins.sbt

```diff
@@ -4,5 +4,5 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts
 addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"        % "3.6.0")
 addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables"    % "2.1.0")
 addSbtPlugin("com.typesafe.play" % "sbt-plugin"            % "2.8.14")
-addSbtPlugin("org.scoverage"     % "sbt-scoverage"         % "1.9.3")
+addSbtPlugin("org.scoverage"     % "sbt-scoverage"         % "2.0.0-M4")
```

Comment:
I would stay away from using M versions as it could mean its not stable and we don't want to use the bleeding edge version.